index.html

```diff
@@ -3102,22 +3116,44 @@ <h3>Verify Proof Sets and Chains</h3>
 Set <var>allProofs</var> to </var><var>securedDocument</var>.<var>proof</var>.
           </li>
           <li>
-For each <var>proof</var> in <var>allProofs</var>, run steps 2 through 11 of the algorithm in
+For each <var>proof</var> in <var>allProofs</var>, do the following steps:
+          </li>
+          <ol class="algorithm">
+            <li>
+Let <var>matchingProofs</var> be an empty list.
+            </li>
+            <li>
+If <var>proof</var> contains a <code>previousProof</code> attribute and that
+attribute is a string, add the element from <var>allProofs</var> with an
+<code>id</code> attribute matching <code>previousProof</code> to matchingProofs.
+If a proof with <code>id</code> does not exist in <var>allProofs</var>, an error
+MUST be raised and SHOULD convey an error type of
+<a href="#MALFORMED_PROOF_ERROR">MALFORMED_PROOF_ERROR</a>.
+If the <code>previousProof</code> attribute is an array, add each element from
+<var>allProofs</var> with an <code>id</code> attribute that matches an element
+of that array. If any matches are missing, an error MUST be raised and SHOULD
```

Comment:
As [above](https://github.com/w3c/vc-data-integrity/pull/224/files#r1417842986).

Reply:
What do you want changed? This has been incorporated in a recent commit.